src/utils/useConnectionOptions/useConnectionOptions.ts

```diff
@@ -17,13 +16,9 @@ export default function useConnectionOptions() {
       video: {
         mode: settings.bandwidthProfileMode,
         dominantSpeakerPriority: settings.dominantSpeakerPriority,
-        renderDimensions: {
-          low: getResolution(settings.renderDimensionLow),
-          standard: getResolution(settings.renderDimensionStandard),
-          high: getResolution(settings.renderDimensionHigh),
-        },
-        maxTracks: Number(settings.maxTracks),
         trackSwitchOffMode: settings.trackSwitchOffMode,
+        contentPreferencesMode: settings.contentPreferencesMode,
```

Comment:
for some reason i'm getting a TS error for `contentPreferencesMode`:
<img width="712" alt="Screen Shot 2021-05-11 at 11 17 01 AM" src="https://user-images.githubusercontent.com/77076398/117840925-90b46e80-b24a-11eb-96b1-7fd2a4077452.png">

but not for `clientTrackSwitchOffControl`? i'm not getting any error when i `npm start` though so it doesn't seem to be an issue. just wondering if you had an idea as to why TS is mad?

Reply:
Thanks for catching it @olipyskoty - this flag was renamed. We need to replace 
`contentPreferencesMode` =>   `clientTrackSwitchOffControl`
everywhere

Reply:
@makarandp0 - We already have `clientTrackSwitchOffControl` everywhere. Should we remove `contentPreferencesMode`? Sounds like we just need to add one new option and not two?

Reply:
@timmydoza weird. I just ran `npm i` again, and now I don't see it either? I think maybe my vscode didn't update after I initially did the install? Sorry for the false alarm!

Reply:
sorry. I misread and mistyped! We do bot `contentPreferencesMode` and `clientTrackSwitchOffControl`.
the rename was 
`subscribedTrackSwitchOffMode` => `clientTrackSwitchOffControl`
which was already done.